helm-service/deploy/service.yaml

```diff
@@ -31,7 +31,7 @@ spec:
         - name: EVENTBROKER
           value: 'http://event-broker.keptn.svc.cluster.local/keptn'
         - name: API
-          value: 'ws://api.keptn.svc.cluster.local'
+          value: 'ws://api-service.keptn.svc.cluster.local:8080'
```

Comment:
I guess this change needs to be done for many other services as well :/

Reply:
Yes, in the dyantrace-service and prometheus-service.